dashboard/urls.py

```diff
@@ -69,6 +80,8 @@
 if 'djangosaml2' in settings.INSTALLED_APPS:
     urlpatterns += (
         url(r'^accounts/', include('djangosaml2.urls')),
+        url(r'^samltest/', echo_attributes),
```

Comment:
login_required for "samltest/"?

Reply:
I can double check, I don't think it will do anything if STUDENT_DASHBOARD_SAML == false because the settings for saml aren't setup in settings.py. This is one path I want to test. We really could probably remove this entirely after we see one time what values it provides on an actual instance.